test/regression/aarch64/Exception.cc

```diff
@@ -151,7 +151,6 @@ TEST_P(Exception, unmapped_sys_reg) {
   EXPECT_EQ(stdout_.substr(0, strlen(err)), err);
 }
 
-#if SIMENG_LLVM_VERSION >= 14
```

Comment:
Is this in response to the SVE vs SVE2 identification issue or something else? I feel like we can keep this sort of checking in

Reply:
Yes - in response to the SVE / SVE2 and SME / SME2 checks (i.e. it is not trivial). This one can stay though, yes